docs/registry/create/_index.md

```diff
@@ -600,67 +631,72 @@ func mainWithArgs(ctx context.Context, args []string, logger logger.Logger) (err
 
-You will also need to create a `requirements.txt` file containing a list of all the dependencies your module relies on.
-For example, a `requirements.txt` file with the following contents ensures that the Viam Python SDK (`viam-sdk`) is installed.
-You may also add additional dependencies as needed:
+1. Create a `requirements.txt` file containing a list of all the dependencies your module relies on.
+   For example, a `requirements.txt` file with the following contents ensures that the Viam Python SDK (`viam-sdk`) is installed:
 
-```sh { class="command-line" data-prompt="$"}
-viam-sdk
-```
+   ```sh { class="command-line" data-prompt="$"}
+   viam-sdk
+   ```
 
-See the [pip `requirements.txt` file documentation](https://pip.pypa.io/en/stable/reference/requirements-file-format/) for more information.
+   Add additional dependencies as needed.
+   See the [pip `requirements.txt` file documentation](https://pip.pypa.io/en/stable/reference/requirements-file-format/) for more information.
 
-The following template sets up a new virtual environment (`venv`), installs the dependencies listed in `requirements.txt`, and runs the module entry point file `main.py`:
+2. Add a shell script that creates a new virtual environment, installs the dependencies listed in `requirements.txt`, and runs the module entry point file `main.py`:
 
-```sh { class="command-line" data-prompt="$"}
-#!/bin/sh
-cd `dirname $0`
+   ```sh { class="command-line" data-prompt="$"}
+    #!/bin/sh
+    cd `dirname $0`
 
-# Create a virtual environment to run our code
-VENV_NAME="venv"
-PYTHON="$VENV_NAME/bin/python"
+    # Create a virtual environment to run our code
+    VENV_NAME="venv"
+    PYTHON="$VENV_NAME/bin/python"
 
-python3 -m venv $VENV_NAME
-$PYTHON -m pip install -r requirements.txt -U # remove -U if viam-sdk should not be upgraded whenever possible
+    python3 -m venv $VENV_NAME
+    $PYTHON -m pip install -r requirements.txt -U # remove -U if viam-sdk should not be upgraded whenever possible
 
-# Be sure to use `exec` so that termination signals reach the python process,
-# or handle forwarding termination signals manually
-exec $PYTHON <your-src-dir-if-inside>/main.py $@
-```
+    # Be sure to use `exec` so that termination signals reach the python process,
+    # or handle forwarding termination signals manually
+    exec $PYTHON <your-src-dir-if-inside>/main.py $@
+   ```
 
-To make your shell script executable, run the following command in your terminal:
+   To make your shell script executable, run the following command in your terminal:
 
-```sh { class="command-line" data-prompt="$"}
-sudo chmod +x <your-file-path-to>/<run.sh>
-```
+   ```sh { class="command-line" data-prompt="$"}
+   sudo chmod +x <your-file-path-to>/<run.sh>
+   ```
+
+See the [Python virtual environment documentation](https://docs.python-guide.org/dev/virtualenvs/) for more information.
```

Comment:
If we retain parallel instructions, recommend changing this See Also to point to https://docs.viam.com/program/python-venv/ instead.